lib/date.js

```diff
@@ -0,0 +1,57 @@
+$('document').ready(function(){
+  var todayDate = moment();
+  $(".current-date").append(todayDate.format("MMMM Do, YYYY"));
+  localStorage.setItem("currentDate", JSON.stringify(todayDate.format("MMMM Do, YYYY")));
+  currentDate = todayDate;
+  add_day();
+
+  $(".day-foreward").on("click", function(){
+    currentDate = currentDate.add(1, 'days');
+    localStorage.setItem("currentDate", JSON.stringify(currentDate.format("MMMM Do, YYYY")));
+    $(".current-date").html(currentDate.format("MMMM Do, YYYY"));
+    add_day();
+  });
+
+  $(".day-backward").on("click", function(){
+    currentDate = currentDate.add(-1, 'days');
+    localStorage.setItem("currentDate", JSON.stringify(currentDate.format("MMMM Do, YYYY")));
+    $(".current-date").html(currentDate.format("MMMM Do, YYYY"));
+    add_day();
+  });
+})
+
+function add_day(){
```

Comment:
@DavidKnott @jdconrad89 definitely get this updated before you submit this project 👍